src/component-library/LoadingSpinner/LoadingSpinner.style.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+import { theme } from 'component-library';
+
+interface BaseLoadingSpinnerProps {
```

Comment:
very minor, but I'm not sure whether we would have props like this, or specific sizes (e.g. `small`, `xl` or whatever). It's fine as it is for now, but might be something we want to consider if and when Jay introduces different use cases.

Reply:
For now, as we do not have any size steps, I'd like to improve it when we have specific discrete steps introduced by Jay.